src/lib/containers/StackChanger.tsx

```diff
@@ -0,0 +1,81 @@
+import { isEqual } from 'lodash-es'
+import * as React from 'react'
+import { useEffect, useState } from 'react'
+import UniversalCookies from 'universal-cookie'
+import { EndpointObject } from '../utils/functions/getEndpoint'
+import { getStyleguideStack } from '../utils/SynapseClient'
+import { STYLEGUIDE_STACK_COOKIE } from '../utils/SynapseConstants'
+import { RadioGroup } from './widgets/RadioGroup'
+
+type Stack = 'production' | 'staging' | 'development'
+const STACK_MAP: Record<Stack, EndpointObject> = {
+  production: {
+    REPO: 'https://repo-prod.prod.sagebase.org',
+    PORTAL: 'https://synapse.org/',
```

Comment:
May as well point to www.synapse.org to avoid the redirector

Reply:
```suggestion
    PORTAL: 'https://www.synapse.org/',
```